app/src/main/java/com/conupods/OutdoorMaps/View/MainActivity.java

```diff
@@ -35,13 +36,25 @@ protected void onCreate(Bundle savedInstanceState) {
 
 
     private void init(){
+        new CountDownTimer(3000, 2000) {
+
+            public void onTick(long millisUntilFinished) {
+                Log.i("DisplaySplash","Displaying Splash Screen");
+            }
+
+            public void onFinish() {
+                startActivity(new Intent(MainActivity.this, MapsActivity.class));
+                overridePendingTransition(R.anim.fade_in, R.anim.slide_out_left);
```

Comment:
Should be a fade out, not slide out . Looks more coherent with other mobile apps

Reply:
I liked the slide though but okay